utils/build.ps1

```diff
@@ -2321,6 +2321,10 @@ function Test-Compilers([Hashtable] $Platform, [string] $Variant, [switch] $Test
         # LLDB Unit tests link against this library
         LLVM_UNITTEST_LINK_FLAGS = "$(Get-SwiftSDK -OS Windows -Identifier Windows)\usr\lib\swift\windows\$($Platform.Architecture.LLVMName)\swiftCore.lib";
       }
+
+      if ($env:ONLY_SWIFT_LLDB_TESTS -eq '1') {
```

Comment:
I really would rather not introduce this type of behaviour in build.ps1. Can we sink this check into lit itself?

Reply:
Would using the `check-lldb-swift` target be better? https://github.com/swiftlang/llvm-project/blob/b6f1a41420ebffc7f043b16080ef0e8afde4d2cc/lldb/test/CMakeLists.txt#L302-L306

we could have `lldb` and `lldb-swift` in the list of tests that build.ps1 supports